ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/support/workspace/LiferayWorkspaceGradleSupport.java

```diff
@@ -69,4 +69,8 @@ public void initBundle() {
 		jobAction.waitForNoRunningJobs();
 	}
 
+	public void prepareGradleWorkspace() {
```

Comment:
should not have method, just write actions in the test case.

Reply:
using this method, we can override it in the @before(), otherwise, all existing tests extend "LiferayWorkspaceGradleSupport" will create workspace with indexSource, but we don't actually need. In this case, what's your plan?

Reply:
we should not use this way to modify the behaviour, we should make disable indexSource by default, and write the special test cases to test enabled indexSource.